### PR TITLE
Add info about local Danger environment variables to Troubleshooting

### PR DESCRIPTION
**Pull request by mtitolo**
This is why I was getting an error trying to run `local`. Totally forgot to add that.

### CODE REVIEW

**Comment by KrauseFx** [comment]
Looks great, thanks @mtitolo 👍

**Comment by KrauseFx** [comment]
I wonder if additionally to the docs update, we should also show a better error message if the request is failing. I'm a big fan of showing instructions right where the user needs them, in this case when the error message is shown

**Comment by mtitolo** [comment]
A better error message would be 💯

Sent from the road




On Thu, Aug 18, 2016 at 5:53 PM -0700, "Felix Krause" <notifications@github.com> wrote:












I wonder if additionally to the docs update, we should also show a better error message if the request is failing. I'm a big fan of showing instructions right where the user needs them, in this case when the error message is shown

File: static/source/guides/troubleshooting.html.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ OK, so, you can use `bundle exec danger local` to have Danger run the last merge
 
 This will run the Danger environment locally, making it possible to iterate and verify syntax etc.
 
+Make sure to setup the `DANGER_GITHUB_API_TOKEN` environment variable before attempting to run `local`. If you are trying to access an Enterprise Github instance, `DANGER_GITHUB_HOST` and `DANGER_GITHUB_API_HOST` must be set.
```
Comment (orta) [inline static/source/guides/troubleshooting.html.md]: Aye this is a good step forwards, ideally we should show examples of what the host/api host look like too

**Comment by orta** [comment]
Local is going to get some significant changes which _should_ be able to let us do better error messages too, as there's less implicit guess-work - https://github.com/danger/danger/issues/382

If either of you want to take a stab at that, you're welcome to, I've got my hands full with maintenance + Danger.JS ATM

**Pull request merged**